src/main/java/models/AuditEvent.java

```diff
@@ -0,0 +1,18 @@
+package models;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
+public class AuditEvent {
+    public String auditEventId;
+    public String source;
+    public String timestamp;
+    public String type;
+    public String workspaceId;
+    public Object actor;
```

Comment:
Returning Object for actor and target is kind of useless. Do we have proper type definitions for these?

Reply:
good call. we have id and email right now! Added classes for actor and target!